ga4gh/repo_manager.py

```diff
@@ -79,6 +80,7 @@ class RepoManager(object):
     vcfIndexExtension = '.vcf.gz.tbi'
     bamExtension = '.bam'
     bamIndexExtension = '.bam.bai'
+    dbExtension = '.db'
```

Comment:
`.db` files represent both Sequence Annotation and RNA data by this point. We'll want to be careful here.